src/ResourceManagement/BatchAI/BatchAIJobImpl.cs

```diff
@@ -129,8 +233,9 @@ public ToolTypeSettings.CognitiveToolkit.Definition.IBlank<BatchAIJob.Definition
         public override async Task<Microsoft.Azure.Management.BatchAI.Fluent.IBatchAIJob> CreateResourceAsync(CancellationToken cancellationToken = default(CancellationToken))
         {
             createParameters.Cluster = new Models.ResourceId(parent.Id);
-            createParameters.Location = Inner.Location;
-            await Manager.Inner.Jobs.CreateAsync(parent.ResourceGroupName, this.Name, createParameters);
+            createParameters.Location = parent.RegionName;
+            var inner = await Manager.Inner.Jobs.CreateAsync(parent.ResourceGroupName, this.Name, createParameters);
+            SetInner(inner);
```

Comment:
Please add cancellation token parameter to async call.

Reply:
@lenala please address a single comment and create a new PR. I'll merge this to see test results tomorrow.